dockerfile

```diff
@@ -1,20 +1,46 @@
 # Install KWIVER to /opt/kitware/kwiver
 # Use latest Fletch as base image (Ubuntu 18.04)
+ARG BASE_IMAGE=kitware/fletch:latest
```

Comment:
This should be changed to: "kitware/fletch:latest-ubuntu18.04-py3"

With the latest changes to the fletch dockerfile, we are now producing different labeling syntax.

The default base fletch image is now "kitware/fletch:latest-ubuntu18.04-py3", I plan on removing "kitware/fletch:latest" altogether.